src/VisualStudio/LiveShare/Test/HandlerShimsTests.cs

```diff
@@ -22,7 +22,7 @@ public void TestLiveShareRequestHandlersExported()
 
             // Verify there are exactly the number of liveshare request handlers as expected.
             // Liveshare shims will verify there is a matching roslyn request handler when they are created.
-            Assert.Equal(21, handlers.Length);
+            Assert.Equal(19, handlers.Length);
```

Comment:
this seems like a strange test.

Reply:
followup PR should delete the rest of the liveshare handlers and this test.